fixtures/tests/briefResponse/specialist.js

```diff
@@ -1,40 +1,37 @@
-import { buyerLogin, sellerLogin } from '../../flows/login/actions';
-import { applyForSpecialist, navigate, selectBrief, viewSpecialistApplication } from '../../flows/opportunities/actions';
-import { create } from '../../flows/brief/specialist';
-import { startBrief } from '../../flows/dashboard/buyer';
-import { respond } from '../../flows/briefResponse/specialist';
+import { buyerLogin, sellerLogin } from '../../flows/login/actions'
+import { applyForSpecialist, navigate, selectBrief, viewSpecialistApplication } from '../../flows/opportunities/actions'
+import create from '../../flows/brief/specialist'
+import startBrief from '../../flows/dashboard/buyer'
+import respond from '../../flows/briefResponse/specialist'
+
+/* eslint-disable no-await-in-loop */
```

Comment:
is this still needed? if so can it be put closer to where the line is with `eslint-disable-next-line`